R/utils_formatters.R

```diff
@@ -740,11 +754,11 @@ num_fmt_factory <- function(context,
     x_str_vals <-
       x_vals %>%
       # Format all non-NA x values with a formatting function
-      format_fn(context) %>%
+      format_fn(context = context) %>%
       # If in a LaTeX context, wrap values in math mode
-      to_latex_math_mode(context) %>%
+      to_latex_math_mode(context = context, active = use_latex_math_mode) %>%
```

Comment:
Use a conditional here inside of `{...}` instead of passing a flag

Reply:
Now done.